sdk/common/AxiosHelper.js

```diff
@@ -1,11 +1,37 @@
-const combineURLs = require("axios/lib/helpers/combineURLs");
-const isAbsoluteURL = require("axios/lib/helpers/isAbsoluteURL");
-const axios = require("axios");
+const { default: axios } = require("axios");
 const querystring = require("query-string");
 const { sign } = require("./RequestSigner");
 const { PDKServerResponseError } = require("./PDKError");
 const FormData = require("form-data");
 
+/**
+ * https://github.com/axios/axios/blob/5b8a826771b77ab30081d033fdba9ef3b90e439a/lib/helpers/isAbsoluteURL.js
+ * Determines whether the specified URL is absolute
+ * @ignore
+ * @param {string} url The URL to test
+ * @returns {boolean} True if the specified URL is absolute, otherwise false
+ */
+function isAbsoluteURL(url) {
+    // A URL is considered absolute if it begins with "<scheme>://" or "//" (protocol-relative URL).
+    // RFC 3986 defines scheme name as a sequence of characters beginning with a letter and followed
+    // by any combination of letters, digits, plus, period, or hyphen.
+    return /^([a-z][a-z\d+\-.]*:)?\/\//i.test(url);
+}
+
+/**
+ * https://github.com/axios/axios/blob/5b8a826771b77ab30081d033fdba9ef3b90e439a/lib/helpers/combineURLs.js
+ * Creates a new URL by combining the specified URLs
+ * @ignore
+ * @param {string} baseURL The base URL
+ * @param {string} relativeURL The relative URL
+ * @returns {string} The combined URL
+ */
+function combineURLs(baseURL, relativeURL) {
+    return relativeURL
+        ? baseURL.replace(/\/+$/, "") + "/" + relativeURL.replace(/^\/+/, "")
+        : baseURL;
+}
+
```

Comment:
Why have we added this? 
Doesn't look like its being used somewhere.

Reply:
It is used here https://github.com/pixelbin-io/pixelbin-js-admin/blob/b3f3503ae8623c63889cb3049b11abe24e6825d2/sdk/common/AxiosHelper.js#L94

Added for compatibility if and when we require signature generation again.